### DISCUSSION TITLE
Blank screen on Vercel deployment

### DISCUSSION THREAD

**Discussion by timB73** [vercel/next.js#59944] [Help] [answered]
### Summary

Hi, I just deployed my Next.js app on Vercel but it gets a blank screen. If I build and run it locally it doesn't do this and works perfectly fine. I looked at the Source tab and the only files listed in the output are favicon.ico, index.html, manifest.json, robots.txt. I use a monorepo but I've already set the root directory to the correct directory.

You can see the deployment here: https://qw-app.vercel.app/

There are no errors in the console and the logs in Vercel are empty. I've deployed other apps in Next.js without issue, this particular one is a migration from Create React App maybe I've missed something...

### Additional information

```js
Operating System:
  Platform: darwin
  Arch: arm64
  Version: Darwin Kernel Version 23.2.0: Wed Nov 15 21:53:18 PST 2023; root:xnu-10002.61.3~2/RELEASE_ARM64_T6000
Binaries:
  Node: 21.4.0
  npm: 10.2.4
  Yarn: 1.22.19
  pnpm: N/A
Relevant Packages:
  next: 14.0.5-canary.27
  eslint-config-next: 14.0.4
  react: 18.2.0
  react-dom: 18.2.0
  typescript: 4.9.4
Next.js Config:
  output: N/A
```

**Comment by timB73** [comment, marked as answer]
Never mind I found the issue. I had a left over index.html inside public folder from Create React App, then I had to set my project settings to Next.js framework as it had defaulted to "Other".

**Reply by wttk1** [reply]
Hi Im running into the same issue, I also have a monorepo, where I set the route to my front end. I also have that remaining index.html file inside the public folder. Did you delete that one? Only difference is that I dont use next.js. 

Thanks a lot for your help!

**Comment by umrashrf** [comment]
In my case, I had to do this https://vercel.com/docs/projects/project-configuration#outputdirectory

**Comment by ahiyantra** [comment]
I faced a similar issue & figured out the solution for that. The localhost version of my website was working without issues for both the "`npm start`" & "`serve -s build`" commands. The vercel version was showing an error related to the "`/build/static/js/main[dot]js`" file's contents due to some kind of configuration & routing issue. It was resolved after adding the following lines to the "`vercel[dot]json`" file:

`  "routes": [    { "src": "/(.*)", "dest": "build/index.html" }  ],`

**Comment by firasbelhiba** [comment]
If you are using VPN ( NordVPN in my case ) and you saw isWhitelisted in your localstorage , kill the nordvpn process ( don't disabled kill it ) and it will work , worked for my case . Vercel Policy I guess

**Reply by enerage** [reply]
Been trying to fix it for 5 hours, and it ended up because of NordVPN, absolutely ridiculous. At least they could have shown an error somewhere..
